frontend/package.json

```diff
@@ -20,8 +20,8 @@
     "prepare": "cd .. && husky frontend/.husky"
   },
   "dependencies": {
-    "@getalby/lightning-tools": "^5.1.2",
     "@getalby/bitcoin-connect-react": "^3.6.2",
+    "@getalby/lightning-tools": "5.1.2",
```

Comment:
Why this change?

Reply:
I did yarn add @getalby/lightning-tools, and it corrected this, it should be in alphabetical order right?

Reply:
it changed it to the exact version, and this was already in the PR (You probably should have just did yarn install?), I think the change should be undone

Reply:
Yeah, but isn't it the convention? To have the packages in sorted order? https://eslint.org/docs/latest/contribute/package-json-conventions#order

Reply:
The order is not the only change, you also changed the version from "^5.1.2" to "5.1.2"